githubissues/main.go

```diff
@@ -98,7 +98,12 @@ func (g *githubissuesNotifier) SendNotification(ctx context.Context, build *cbpb
 		return nil
 	}
 
-	webhookURL := fmt.Sprintf("%s/%s/issues", githubApiEndpoint, g.githubRepo)
+	repo := GetGithubRepo(build)
```

Comment:
Can this be considered a change of behavior? If the notification repo is intentionally set different as the build source repo?

Reply:
Yes it might create a different expectation as I commented above. 

> Pushed a new commit to skip notification if cannot infer repository from build information, as I think this is more intuitive than just creating notification on the config repo in yaml file
> However, this also means it renders the repo configuration useless

However, for this specific notifier, I don't see a reason why notification repo should be set different from the build source repo. I can imagine one might try to keep track of build issues in separated repo, but then what's the point of doing so?